frontend/src/services/QuestionService.js

```diff
@@ -3,29 +3,26 @@ import { QUESTION_SERVICE_ENDPOINT } from "../constants";
 const axios = require("axios");
 
 export async function getEasyQuestion(room_id) {
```

Comment:
Can we just have a single method and pass in the difficulty as a parameter to the method instead of duplicating?

Reply:
Alright! I've made the changes!